templates/jail.local.j2

```diff
@@ -0,0 +1,231 @@
+# {{ ansible_managed }}
+
+#
+# WARNING: heavily refactored in 0.9.0 release.  Please review and
+#          customize settings for your setup.
+#
+# Changes:  in most of the cases you should not modify this
+#           file, but provide customizations in jail.local file,
```

Comment:
In general, comments that explain how one would manually write this file can be removed. The correct way to populate it is using the role variables. The file shouldn't be edited by hand and if one needs hints on how to configure it, the role README should give them.